scripts/run-dev.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cp -f ~/work/lndr/lndr-backend/data/lndr-server.dev.config ~/work/lndr/lndr-backend/data/lndr-server.config
+
+lndr-server
```

Comment:
I'm not sure whether this file should be committed at all. If there are issues with the Docker environment w/docker-compose, then either (a) let's address those issues, or (b) let's provide a holistic alternative for bootstrapping a development environment from scratch on the host. I don't have strong opinions on the approach, but I do think it's important that we have a way to bootstrap a development environment from scratch quickly and easily, whatever that is, and for that environment to work well for us.

IIRC, the Docker-based environment works reasonably well for spinning up a local server if you're not actively making changes to it (e.g: developing the mobile app), but can become painful if actively developing on the server.

I'd propose that, in the meantime, refrain from committing this file (or move it to a helper script on your machine outside of this project), and we'll figure out a path to better automation around dev environment bootstrapping and management.